imports/node-app/devserver/schemas.js

```diff
@@ -15,6 +15,9 @@ import paymentsExample from "/imports/plugins/included/payments-example/server/n
 import paymentsStripe from "/imports/plugins/included/payments-stripe/server/no-meteor/schemas";
 import shippingRates from "/imports/plugins/included/shipping-rates/server/no-meteor/schemas";
 
+// UNIT TEST
+import remoteGraphQL from "/imports/plugins/unit-test/remote-graphql/server/no-meteor/schemas";
```

Comment:
This will someday be the actual app, and not just for tests. In general I try to keep it working as an alternative to running the Meteor app. So it would be good if we can find a way to not import test schemas into it.

Perhaps you can extend it where it's used in the `TestApp` constructor? If you take `additionalSchemas` array as a constructor option, then you can pass that in during the test and keep it out of non-test code.